uwh-refbox/src/main.rs

```diff
@@ -140,53 +140,259 @@ fn main() -> std::result::Result<(), Box<dyn std::error::Error>> {
+                        .translate(Point::new(74, 18)),
+                );
+                match state.current_period {
+                    GamePeriod::BetweenGames => display.draw(
+                        Font8x15::render_str("NEXT GAME")
+                            .stroke(Some(game_color))
+                            .translate(Point::new(92, 2)),
+                    ),
+                    GamePeriod::FirstHalf => display.draw(
+                        Font8x15::render_str("1ST HALF")
+                            .stroke(Some(game_color))
+                            .translate(Point::new(100, 2)),
+                    ),
+                    GamePeriod::HalfTime => display.draw(
+                        Font8x15::render_str("HALF-TIME")
+                            .stroke(Some(game_color))
+                            .translate(Point::new(92, 2)),
+                    ),
+                    GamePeriod::SecondHalf => display.draw(
+                        Font8x15::render_str("2ND HALF")
+                            .stroke(Some(game_color))
+                            .translate(Point::new(100, 2)),
+                    ),
+                    GamePeriod::PreOvertime => display.draw(
+                        Font8x15::render_str("PRE-OVERTIME")
+                            .stroke(Some(game_color))
+                            .translate(Point::new(80, 2)),
+                    ),
+                    GamePeriod::OvertimeFirstHalf => display.draw(
+                        Font8x15::render_str("O/T 1ST HALF")
+                            .stroke(Some(game_color))
+                            .translate(Point::new(80, 2)),
+                    ),
+                    GamePeriod::OvertimeHalfTime => display.draw(
+                        Font8x15::render_str("O/T HALF TIME")
+                            .stroke(Some(game_color))
+                            .translate(Point::new(76, 2)),
+                    ),
+                    GamePeriod::OvertimeSecondHalf => display.draw(
+                        Font8x15::render_str("O/T 2ND HALF")
+                            .stroke(Some(game_color))
+                            .translate(Point::new(80, 2)),
+                    ),
+                    GamePeriod::PreSuddenDeath => display.draw(
+                        Font8x15::render_str("PRE-SUDDEN DEATH")
+                            .stroke(Some(game_color))
+                            .translate(Point::new(64, 2)),
+                    ),
+                    GamePeriod::SuddenDeath => display.draw(
+                        Font8x15::render_str("SUDDEN DEATH")
+                            .stroke(Some(game_color))
+                            .translate(Point::new(80, 2)),
+                    ),
+                    _ => display.draw(
+                        Font8x15::render_str("ERROR_1")
+                            .stroke(Some(game_color))
+                            .translate(Point::new(64, 2)),
+                    ),
```

Comment:
I think we don't want a default case in this match. If you remove it, the compiler will check that you've covered all the cases, which we want here

Reply:
oh so the compiler can recognize that all options are listed?

Reply:
yup